parser/Parser/Parser.py

```diff
@@ -0,0 +1,174 @@
+from neo4j import GraphDatabase
+import re,unicodedata,unidecode
+
+
+class Parser:
+
+    def __init__(self,filename,uri="bolt://localhost:7687"):
+        self.driver = GraphDatabase.driver(uri)
+        self.session = self.driver.session() #Doesn't create error even if there is no active database
+        self.filename=filename
+
+    def file_name(self):
+        """open the file filename and return the list of entries, each entry is a list a the sanitized lines"""
+        self.filename = self.filename + ( '.txt' if (len(self.filename)<4 or self.filename[-4:]!=".txt") else '')  #to not get error if extension is missing
+
+    def file_iter(self,start=0):
+        """generator to get the file line by line"""
+        counter=0
+        with open(self.filename,"r",encoding="utf8") as file:
+            for line in file:
+                if counter < start : counter+=1
```

Comment:
for this kind of check it's ok to use a "continue" statement and avoid the else:

```
if counter < start:
    counter += 1
    continue
```

even better you can use itertools.islice:

```
for line in itertools.islice(file, start=start):
```